Triangulation_3/examples/Triangulation_3/CMakeLists.txt

```diff
@@ -1,29 +1,11 @@
-# Created by the script cgal_create_cmake_script
-# This is the CMake script for compiling a CGAL application.
+project( Triangulation_3_Examples )
 
-
-project( Triangulation_3_example )
-
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 2.8.12)
 
 
 find_package(CGAL QUIET)
 
 if ( CGAL_FOUND )
-
-  include( ${CGAL_USE_FILE} )
-  
-  find_package( TBB QUIET )
-
-  if( TBB_FOUND )
-    include(${TBB_USE_FILE})
-    list(APPEND CGAL_3RD_PARTY_LIBRARIES ${TBB_LIBRARIES})
-  endif()
-
-  include( CGAL_CreateSingleSourceCGALProgram )
```

Comment:
Actually you are quite right. If we want the CMakeLists.txt to be compatible with:
  -  CGAL in pure header only,
  - or header only step 1,
  - or CGAL precompiled,
then I have to modify the other `CGALConfig.cmake` (those that are generated), so that `CGAL_CreateSingleSourceCGALProgram` is included automatically.